understanding/20/use-of-color.html

```diff
@@ -36,11 +36,12 @@ <h2>Intent of Use of Color</h2>
          of prompting a response would be: using highlighting on form fields to indicate that
          a required field had been left blank.
       </p>
+
+      <p>Contrast between colors can be considered an alterantive method of conveying information.  Color (generally defined as combination of hue, saturation and luminance) can be differentiated if there is a sufficient difference in contrast. If the contrast ratio (<a>relative luminance</a>) between the colors differ by at least 3:1, or if there is another method of conveying that information it would pass this criterion. For example, a text link that only differs from adjacent text using color and there is no other visual indication that the text is linked (e.g. the link underline is removed) needs to ensure that the link color meets the 3:1 luminosity contrast ratio relative to the non-linked text color in order to meet this Success Criteria.</p>>
```

Comment:
"alterantive" > "alternative"

i'd add commas and a "where"

"only differs from adjacent text using color and there is no other visual indication that the text is linked (e.g. the link underline is removed) needs to ensure" > "only differs from adjacent text using color*,* and *where* there is no other visual indication that the text is linked (e.g. the link underline is removed)*,* needs to ensure"

Reply:
As I went through the other issues, I think the explanation is covered in the changes from #1500, so I'll remove that.